in_toto/in_toto_run.py

```diff
@@ -312,4 +313,11 @@ def main():
         LOG.error("(in-toto-run) %s: %s", type(e).__name__, e)
         sys.exit(1)
 
+    if (
+        link
+        and getattr(link, "signed", None)
+        and link.signed.byproducts.get("return-value")
+        and link.signed.byproducts.get("return-value") != 0
```

Comment:
Is there a reason, you're not just returning this 0 but the hardcoded one below?